mcp/commanding/commands.c

```diff
@@ -348,6 +348,87 @@ void SingleCommand(enum singleCommand command, int scheduled)
             CommandData.if_power.relay_10_off = 1;
             CommandData.if_power.update_pbob = 1;
             break;
+        // motor pbob
+        case motor_pbob_relay_1_on:
+            CommandData.of_power.relay_1_on = 1;
```

Comment:
Should these sub-structs be e.g. `CommandData.motor_power.relay_X_on`?

Reply:
Good catch, thought I had swapped the substruct as well